projects/transformers/run.py

```diff
@@ -299,12 +299,13 @@ def run_finetuning_single_task(
     eval_dataset = tokenized_datasets[
         "validation_matched" if data_args.task_name == "mnli" else "validation"
     ]
+
     test_dataset = None
-    if ((data_args.task_name is not None or data_args.test_file is not None)
-       and training_args.do_predict):
-        test_dataset = tokenized_datasets[
-            "test_matched" if data_args.task_name == "mnli" else "test"
-        ]
+    if (data_args.task_name is not None or data_args.test_file is not None):
```

Comment:
I didn't understand this change, what is the difference between the old version and the new?

Reply:
No semantic difference. I just couldn't figure out how to make it pep compliant. I was getting issues about the indent, or the length of the line, so just broke it up.